### PR TITLE
Update time.py (for real this time)

### PR DESCRIPTION
**Pull request by LegitSi**
sorry but i had several skill issues, this SHOULD be fixed now

### CODE REVIEW

File: cogs/time.py [hunk 1]
```diff
@@ -27,12 +27,12 @@ async def time(self, interaction: discord.Interaction):
...
-      if datetime.utcfromtimestamp().strftime('%-d %B %Y %H:%M:%S'):
-          readable = datetime.utcfromtimestamp().strftime('%-d %B %Y %H:%M:%S')
+      try:
+          readable = (datetime.utcfromtimestamp(0)+timedelta(seconds=currentrp)).strftime('%d %B %Y %H:%M:%S')
           embed = embed_template("The current date and time is:", readable)
           await interaction.response.send_message(embeds=[embed])
-      except e as Exception:
-          error_embed = error_template(f"Conversion to readable time failed. Fuck.\n{logging.log.error(e)}")
+      except Exception as e:
+          error_embed = error_template(f"Time command failed.\n{logging.log.error(e)}")
```
Comment (infernostars) [inline cogs/time.py]: the logging.log stuff will log it out to the console, so seperate it off
also logging.log.exception() on its own will work [not your fault, me being dumb]

Comment (LegitSi) [reply cogs/time.py]: real

**Comment by infernostars** [comment]
why are you putting it in the fstring still, it doesnt return anything, it just logs it
it should be a seperate line

**Comment by infernostars** [comment]
its not perfect but /shrug

**Pull request merged**